src/styles/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  padding: 50px 40px 72px;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: getCatalogWidth($card-width,$cards-in-row,$distance-between-cards);
+  margin: 0 auto;
```

Comment:
In BEM blocks don't dictate their position. Add proper styles to the parent element.